src/modules/did/did.js

```diff
@@ -795,6 +796,47 @@ class DIDModule {
       }
     });
 
+    // Fetch BBS+ keys if needed
+    if (getBbsPlusSigKeys === true) {
```

Comment:
shouldnt need `getBbsPlusSigKeys === true` just `if (getBbsPlusSigKeys)`

Reply:
Wanted to be explicit